S23/pmirukula/Conversion.java

```diff
@@ -0,0 +1,44 @@
+import java.util.*;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.*;
+import java.util.Locale;
+
+public class Conversion {
+	public void conversionlogic() 
+	{
+	}
+

```

Comment:
// Is this function used somewhere ? Do we need it?

Reply:
I had originally intended to write a method rather than writing the entire thing in the main, but I was short on time. I'll discontinue using the technique.
Thank you

Reply:
Writing this as a main function only is not an ideal design choice, since the function cannot then be used as library function in other code.  I'd refactor into its own re-usable class.